src/js/libgeo/Prover.js

```diff
@@ -45,6 +45,9 @@ function applyIncidence(a, b) {
 
 function incidentPL(p, l) {
     return {
+        getInvolved: function() {
+            return [p, l];
+        },
```

Comment:
If I didn't know better, I'd read the name here as an invitation for possible future contributors: *Get involved with the CindyJS project!* :smile: